apis/v1/shared_types.go

```diff
@@ -334,9 +334,11 @@ const (
 	RouteReasonNoMatchingListenerHostname RouteConditionReason = "NoMatchingListenerHostname"
 
 	// This reason is used with the "Accepted" condition when there are
-	// no matching Parents. In the case of Gateways, this can occur when
-	// a Route ParentRef specifies a Port and/or SectionName that does not
-	// match any Listeners in the Gateway.
+	// no matching Parents. In the case of Gateways, this can occur when:
+	// * the route ParentRefs field is empty
```

Comment:
if there are no parent refs, can an implementation set any status at all? since status is per parentRef -- https://gateway-api.sigs.k8s.io/reference/spec/#gateway.networking.k8s.io/v1.RouteParentStatus ?

I wonder if this comment necessary

Reply:
you are right 😮‍💨 removed that point.